src/figureplotting.jl

```diff
@@ -18,17 +18,11 @@ function is_plot_3d(p::PlotFunc, args...)
 
     # Otherwise, we check the arguments
     non_obs = to_value.(args)
-    conv = try
-        convert_arguments(p, non_obs...)
-    catch e
-        if e isa MethodError
-            conv = non_obs
-        else
-            rethrow(e)
-        end
-    end
-
-    Typ, args_conv = apply_convert!(p, Attributes(), conv)
+    RealP = plottype(p, non_obs...)
+    result = is_plot_type_3d(RealP)
+    isnothing(result) || return result
```

Comment:
I think 
```julia
if result !== nothing
    return result
end
```
is clearer, this kind of confuses me

Reply:
I feel like the boat has sailed for this one :D 
I've actually started to use `isnothing` consistently and removed all `=== nothing` instead.
`|| return x`  is debatable, i got confused by it as well in the beginning but if you use it more it's kind of nice.
If we remove it here, we should remove it in the whole code base...Maybe at some point we should introduce some style guidelines?